### DISCUSSION TITLE
Sending BLE to navigate gallery view when using Media Mod

### DISCUSSION THREAD

**Discussion by thelifeaquatic** [gopro/OpenGoPro#96] [Q&A]
Hi all,
Am working on a Pi/python based system, which is working well, except when using the media mod. When powering up, the camera starts up in gallery mode, and must be navigated using either the physical camera buttons, or the smart remote.

How can I replicate the BLE commands sent by the smart remote to navigate this gallery screen? On the smart remote, the "mode" and "shutter" buttons are used to scroll and select, but the usual BLE commands for those buttons (eg start recording) do not work when in gallery view. 

Any help much appreciated

Cheers

**Comment by KonradIT** [comment]
Have you tried setting it to photo mode/video mode via BLE?

**Reply by thelifeaquatic** [reply]
no, but I have tried using the "start record" BLE command, because the shutter button on the smart remote acts as the select command, but that did not do anything. So I assumed there must be a more specific command for when the camera is in gallery mode.
Or maybe you mean that perhaps by sending the mode command, it might exit gallery mode and go direct to video mode?? I will give that a try
